pkg/webreg/webreg.go

```diff
@@ -498,6 +498,28 @@ either:
   </li>
 </ul>
 
+<h4 id="buildroot"><a href="#buildroot">Build Root Image</a></h4>
+<p>
+The build root image must contain all dependencies for building executables and
```

Comment:
So originally I was going to ask you to explain what the build root was in the first place, since it's not really introduced here - then I realized that the sentences that explain it and introduce the concept already exist, a paragraph below, so it seemed best to first introduce it, then explain the options. I think what I want is that this explanation comes after an introduction so someone reading top-down understands what we're talking about. At this point in the doc we have not yet even talked about the artifacts we build or introduced the build root as a concept so it is confusing. If you want to keep it up here and just give it the introduction it needs that's fine, or move it down (and keep the header) after the current introduction.

Reply:
I'm not sure I follow. The build root is already mentioned in the paragraph above ("Configuring inputs") - I used the definition there verbatim and just elaborated on it by describing the other (first one is the import) way how it can get into a ci-operator run.

What exact paragraph you mean by "the introduction" of the build root? The _Starting FROM the image described as the build_root..._ one? Because to me that paragraph does not deal with build root at all, just mentions it. Would adding a sentence or two why the build root image is necessary to my new section achieve what you want?

Reply:
I'm dumb, did not scroll up high enough